src/request-logger.ts

```diff
@@ -4,7 +4,7 @@ export default function attachSuperagentLogger (logger, data, req) {
   let method = req.method;
   logger = logger || console;
 
-  logger.debug(`request: ${method.toUpperCase()} ${req.url}`, { timestamp, data });
+  logger.debug(`request: ${method.toUpperCase()} ${req.url}`, { timestamp, data }, true);
```

Comment:
I like to be safe with debug logs incase the logging is turned up and debug starts getting sent to the server.